octopuslite/reader.py

```diff
@@ -159,16 +177,32 @@ def _parse_files(self):
         sample = io.imread(files[0])
         self._shape = sample.shape if self._crop is None else self._crop
 
-        channels = {k:[] for k in Channels}
-
-        # parse the files
-        for f in files:
-            channel = parse_filename(f)['channel']
-            channels[channel].append(f)
+        channels = {k: [] for k in Channels}
+
+        # remove blank frames and parse files
+        if self._remove_blank_frames is not None:
+            if isinstance(self._remove_blank_frames, tuple):
```

Comment:
Again, what happens if it's not a tuple? Why do you need to check this here?

Reply:
I originally had it as another solution to having transformations applied on cropped image sets but i now remove the images in the first instance so i'm going to remove this as its time consuming and unnecessary